tests/test_calculator.hh

```diff
@@ -637,8 +650,15 @@ namespace rascal {
       // Count all the keys in the sparse gradient structure where the gradient
       // is nonzero (i.e. where the key has an entry in the structure)
       for (auto neigh : center) {
+        // TODO(max,felix) there should be a way to write "if neigh is ghost"
```

Comment:
What do we want to do about this TODO? Convert it to an issue, or is it a simple fix to implement this function (`is_ghost()`) in StructureManager?